VocaDbWeb/Views/Event/ViewSeriesVersion.cshtml

```diff
@@ -0,0 +1,55 @@
+@using VocaDb.Web.Helpers;
+@inherits VocaDb.Web.Code.VocaDbPage<VocaDb.Web.Models.Shared.ViewVersion<VocaDb.Model.DataContracts.ReleaseEvents.ArchivedEventSeriesVersionDetailsContract>>
+
+@{
+
+	PageProperties.Title = "Revision " + Model.Entry.ArchivedVersion.Version + " for " + Model.Entry.Name;
+
+	if (Model.Entry.ReleaseEventSeries != null) {
+		ViewBag.Parents = new[] {
+			Html.ActionLink("Events", "Index"),
+			Html.ActionLink(Model.Entry.ReleaseEventSeries.Name, "SeriesDetails", new { id = Model.Entry.ReleaseEventSeries.Id }),
+			Html.ActionLink("Revisions", "SeriesVersions", new { id = Model.Entry.ReleaseEventSeries.Id })
+		};
+	} else {
+		ViewBag.Parents = new[] {
+			Html.ActionLink("Events", "Index"),
+		};
+	}
+
+}
+
+@if (Model.Entry.ComparableVersions.Any())

```

Comment:
It's a small thing, but the braces here are different than elsewhere. I'll try to add these to .editorconfig for Visual Studio at least.

Reply:
fixed